x-pack/plugin/security/src/test/java/org/elasticsearch/test/SecuritySettingsSource.java

```diff
@@ -101,7 +103,12 @@ public SecuritySettingsSource(int numOfNodes, boolean sslEnabled, Path parentFol
         this.subfolderPrefix = scope.name();
         this.sslEnabled = sslEnabled;
         this.hostnameVerificationEnabled = randomBoolean();
-        this.usePEM = randomBoolean();
+        // Use PEM instead of JKS stores so that we can run these in a FIPS 140 JVM
+        if (Security.getProviders()[0].getName().toLowerCase(Locale.ROOT).contains("fips")) {
```

Comment:
I'd like to limit the number of places we have this. Can usePEM be passed in as a variable and we use isInFipsJvm() externally? We could also make isInFipsJvm static (if it is not already)

Reply:
Agreed, this was the only one instance of having this outside `inFipsJvm()` which felt ok, but there's no reason not to import `inFipsJvm()` ( it's already static )